google-cloud-pubsub/lib/google/cloud/pubsub/subscriber.rb

```diff
@@ -66,35 +64,34 @@ module PubSub
       class Subscriber
         include MonitorMixin
 
-        attr_reader :subscription_name, :callback, :deadline, :streams,
-                    :inventory, :message_ordering, :callback_threads,
+        attr_reader :subscription_name, :callback, :deadline, :streams, :message_ordering, :callback_threads,
                     :push_threads
 
         ##
         # @private Implementation attributes.
-        attr_reader :stream_inventory, :stream_pool, :thread_pool, :buffer,
-                    :service
+        attr_reader :stream_pool, :thread_pool, :buffer, :service
 
         ##
         # @private Create an empty {Subscriber} object.
-        def initialize subscription_name, callback, deadline: nil,
-                       message_ordering: nil, streams: nil, inventory: nil,
+        def initialize subscription_name, callback, deadline: nil, message_ordering: nil, streams: nil, inventory: nil,
                        threads: {}, service: nil
           @callback = callback
           @error_callbacks = []
           @subscription_name = subscription_name
           @deadline = deadline || 60
-          @streams = streams || 4
-          @inventory = inventory || 1000
+          @streams = streams || 2
+          @inventory = inventory
+          @inventory = { limit: @inventory } unless inventory.is_a? Hash
+          @inventory[:limit] = Integer(@inventory[:limit] || 1000)
+          @inventory[:bytesize] = Integer(@inventory[:bytesize] || 100_000)
```

Comment:
Should this be `100_000` (100KB) or `100_000_000` (100MB)?

Reply:
Fixed.